doc/modules/kernel_ridge.rst

```diff
@@ -0,0 +1,55 @@
+.. _kernel_ridge:
+
+===========================
+Kernelized ridge regression
+===========================
+
+.. currentmodule:: sklearn.kernel_ridge
+
+Kernelized ridge regression (KRR) combines ridge regression (linear least
+squares plus l2-norm regularization) with the kernel trick. It thus
+learns a linear function in the space induced by the respective kernel and
+the data. For non-linear kernels, this corresponds to a non-linear
+function in the original space.
+
+The model learned by :class:`KernelRidge` is identical to support vector regression
+(:class:`SVR`). However, different loss functions are used (ridge versus
```

Comment:
you are right; the term ridge loss used in some presentations on ridge regression  (e.g., http://statweb.stanford.edu/~nzhang/203_web/lecture12_2010.pdf, slide 6) doesn't make sense. I will change that part